Example/EmissionTests/TestApps.js

```diff
@@ -1,24 +1,21 @@
-/* @flow */
-'use strict';
-
-import Emission from 'emission';
-import React from 'react';
-import ReactNative from 'react-native';
-const { View } = ReactNative;
-const { TestModule } = ReactNative.NativeModules;
+import Emission from 'emission'
+import React from 'react'
```

Comment:
Yeah that's correct, still working on converting all the tests. I'll have it updated soon